### DISCUSSION TITLE
Error: Wallbox: charge power: outdated!

### DISCUSSION THREAD

**Discussion by gvisie** [sweber/wallbox-mqtt-bridge#2] [Announcements]
@sweber  Is this a problem for the functionality of evcc?

**Comment by sweber** [comment]
Hi,
Sorry for the late reply. Somehow gitlab didn't notify me.

About your issue. 
It depends. Normally the walbox will update the topic `charging_power` with the currently used power. However if your charge for a longer time with absolute constant power the topic will not be updated since it doesn't change.

If your experience persistent problems with that behaviour i'd suggest that your add the `charging_power` topic to the `interval_updated_topics` in your mqtt bridge configuration. 
Thats exactly the purpose of that property. This makes sure that the value of that topics don't get stale even if the value itself doesn't change thus populating the same value in the specified intervall.

And yes this might lead to issues in evcc since it has a mechanism implemented that times out the data of the charger and thus no longer knows the correct state of it. Its a bit of a pain point that can be circumvented with above solution.

Let me know if your need some more help.
